recursio-client/Managers/GameManager.gd

```diff
@@ -1,82 +1,69 @@
 extends Node
 class_name GameManager
 
+
+onready var _round_manager: RoundManager = get_node("../RoundManager")
+
+
 # These are optional
-var _game_result_screen
+var _game_end_screen
 var _countdown_screen
 
 var _level: Level
+var _player: Player
+var _enemy: Enemy
 
-var _team_id := -1
-var _countdown_time: float = Constants.get_value("gameplay", "countdown_phase_seconds")
+var _player_kills: Array = [0,0]
+var _player_deaths: Array = [0,0]
+var _ghost_kills: Array = [0,0]
+var _ghost_deaths: Array = [0,0]
 
-func _ready():
-	if has_node("../../GameResultScreen"):
-		_game_result_screen = get_node("../../GameResultScreen")
+
+func _ready() -> void:
+	if has_node("../../GameEndScreen"):
+		_game_end_screen = get_node("../../GameEndScreen")
```

Comment:
This is something you or Karl implemented. There was a comment above the _game_end_screen declaration saying they are optional, so I didn't want to presume. I think this is because the Tutorials might not have the two screens.

Reply:
Ah true, makes sense
Nvm then